docs/pages/blog/mui-v5.md

```diff
@@ -0,0 +1,212 @@
+    MuiButton: {
+      variants: [
+```
+
+```jsx
+import { createMuiTheme, Button } from '@material-ui/core';
+
+// 1. Extend the theme.
+const theme = createMuiTheme({
+  palette: {
+    neutral: {
+      main: '#5c6ac4',
+    },
+  },
+});
+
+// 2. Notify TypeScript about the new color in the palette
+declare module '@material-ui/core/styles' {
+  interface Palette {
+    neutral: Palette['primary'];
+  }
+  interface PaletteOptions {
+    neutral: PaletteOptions['primary'];
+  }
+}
+
+// 3. Profit
+<Button color="neutral"  />
+```
+
+### Global class names
+
+- Explain the problem, the solution
+
+### Hook only components
+
+- Explain the problem, the solution: isolate the logics of the existing components into standalone hooks).
+
+## Support of multiple adapters
+
+- styled-components
+- ask for help on goober https://github.com/mui-org/material-ui/pull/27776
+
+## New components
+
+### Improved Grid
+
+- Covered a bit in https://material-ui.com/blog/2021-q2-update/
+
+### Promotion of components from the Lab to the Core
+
+- Covered a bit in https://material-ui.com/blog/2020/
+
+### Stack
+
+- Covered a bit in https://material-ui.com/blog/2021-q1-update/
+
+### LoadingButton
+
+- Covered a bit in https://material-ui.com/blog/2020-q2-update/
+
+### TrapFocus
+
+- Covered a bit in https://material-ui.com/blog/2020-q3-update/.
+
+### Date pickers
+
+- Transfer of ownership from third-party to mui-org.
+
+### More Material Design icons
+
+- Covered a bit in https://material-ui.com/blog/2020-q2-update/, https://material-ui.com/blog/2021-q1-update/
+
+### Data Grid
+
+- See next section
+- Covered a bit in https://material-ui.com/blog/2020-q3-update/
+
+## A new product line: X
+
+- What are the new product separation
+- What this product line is about
+- Why does it even exist
+- Present the new Data Grid component, hopefully, released as v5.0.0 at the same time.
+
+## Change on the supported platforms
+
+- Updated TypeScript version
+- Updated node.js version
+- Updated React's version
+- Updated browsers support. Stop all work on IE 11, but specific IE 11 will only be removed in v6.
+  Moving IE 11 to a different bundle has allowed saving [-6kB](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216) overall.
+
+## Improved DX
+
+### Smaller demos in the docs
+
+- Covered a bit in https://material-ui.com/blog/2021-q1-update/
+
+### Props descriptions in IntelliSense
+
+- Already a bit covered in https://material-ui.com/blog/2020-q2-update/
+
+### Migration from Enzyme to Testing Library
+
+- The value? Makes it easier for developers to copy the source, and maintainers to make sure the components are easy to test
+
+### TypeScript migration
+
+- Share our progress https://github.com/mui-org/material-ui/issues/15984
+- Covered a bit in https://material-ui.com/blog/2020-q3-update/
+- The value? Helps keeping the types as accurate as possible
+
+### Strict Mode support
+
+- Docs, test, etc. now runs in StrictMode
```

Comment:
Also add a note that we're already React 18 StrictMode compliant and run nightly integration tests to ensure continuous compliance.